src/Components/AuthCart/index.js

```diff
@@ -0,0 +1,56 @@
+import React from "react";
+import "./styles.css";
+
+import CartCard from "../CartCard";
+
+function AuthCart() {
+  const arr = [1, 2];
+  return (
+    <div className="cart-style">
+      <h1>My Cart (2)</h1>
+
+      {/* ------------------ */}
+      <div class="container-cart flex-r">
+        {/* <!-- items --> */}
+
+        <div className="flex-c">
+          {arr.map((item) => (
+            <CartCard key={item._id} item={item} />
+          ))}
+        </div>
+
+        {/* <!-- order --> */}
+        <div class="order border-radius-S flex-c">
```

Comment:
`class` can be replaced with `className`.

Reply:
Thank you for pointing it out i will fix that